examples/dial/dial.slint

```diff
@@ -11,11 +11,7 @@ export global AppState {
     in-out property <length> elementRadius: 185px;
 
     pure public function normalizeAngle(angle: angle) -> angle {
-        if angle >= 0 {
-            return Math.mod(angle, 360deg);
-        } else {
-            return 360deg + Math.mod(angle, 360deg);
-        }
+        return (angle + 360deg).mod(360deg);
```

Comment:
```suggestion
        return (angle + 360deg).mod(360deg);
```

Isn't this a no-op? :)

Reply:
Knowing me, possibly. However atan2 returns an angle between -180 to 180. Then unlike awesome languages such as javascript, mod in slint can return a negative number. So the +360deg ensures its a positive number. Then its also used to normalise the first value when you put your finger on the dial. Say the dial is already at 45 degrees the dial records that and subtracts the value of where you places your finger on the dial. Again possibly creating a negative value. So this ensure everything is always between 0 and 359. Did I get it right?

Reply:
Thank you, this is a crucial detail I was missing. You’re right:)

Reply:
honestly, I think we may consider changing this behavior (of mod)
I can't think of any practical reason to return a negative number